### PR TITLE
fix(create-frog): `next` template incorrect onboarding path

### PR DESCRIPTION
**Pull request by dalechyn**

### CODE REVIEW

File: create-frog/create.ts [hunk 1]
```diff
@@ -10,6 +10,38 @@ const __dirname = dirname(fileURLToPath(import.meta.url))
...
+    return '/api'
+  if (
+    name === 'bun' ||
+    name === 'node' ||
+    name === 'cloudflare-worker' ||
+    name === 'default' ||
+    name === 'vercel'
+  )
+    // frog instance is run via `frog dev`
+    return '/dev'
```
Comment (vikiival) [inline create-frog/create.ts]:
```suggestion
  switch(name) {
    case 'next':
      return '/api'
    default:
      return '/dev'
  }
```

Comment (dalechyn) [reply create-frog/create.ts]: > 

I still believe it is important to throw on unknown template name as such might have a different port and endpoint.
But will refactor to switch clause!

**Comment by megabyte0x** [comment]
Is this the fix for the bug causing issue in running frog dev in the next template?

**Comment by dalechyn** [comment]
> Is this the fix for the bug causing issue in running frog dev in the next template?

Nope, it fixes create-frog emitting incorrect outout

File: create-frog/create.ts [hunk 2]
```diff
@@ -10,6 +10,36 @@ const __dirname = dirname(fileURLToPath(import.meta.url))
...
+
+function templateNameToDefaultFrogPath(name: string): string {
+  switch (name) {
+    case 'next':
+      return '/api'
+    case 'bun':
+    case 'node':
+    case 'cloudflare-worker':
+    case 'default':
+    case 'vercel':
```
Comment (tmm) [inline create-frog/create.ts]: the vercel template uses `/api` too

File: .changeset/clean-houses-fail.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+---
+"create-frog": patch
+---
+
+Fixes incorrect url shown for `next` template at the completion of `create-frog`.
```
Comment (tmm) [inline .changeset/clean-houses-fail.md]:
```suggestion
Fixed incorrect url shown for `next` and `vercel` templates at the completion of `create-frog`.
```

**Review by tmm — CHANGES_REQUESTED** [review]

**Pull request merged**